### ISSUE TITLE
Streamline genre searching / eliminate search by sub genre

### ISSUE THREAD

**Issue by zpine1** [zpine1/deejays#2]
Eliminate the “Sub genre” search box; keep the Genre search using the dropdown, but have it return all records with that genre value in either Genre or Sub genre fields.

**Comment by dancinglightning** [comment]
i checked, there are over 700 unique genres.
Searching them by a list of 20 will yield very bad results.

Maybe when the genre data is cleaned up we can return to this.

**Comment by zpine1** [comment]
Using the last excel export of the db, which has 2256 rows, I did a case-insensitive search to see how many songs genre field contained each of the 26 genre categories. (I also checked and got similar results using the app and searching only using the subgenre search and typing in the category words):

afrobeat 6
alternative 58
ambient 37
balkan 4
blues 43
classical 26
country 8
dance 142
devotional 8
electronic 313
electroswing 3
folk 71
funk 99
hip hop / rap 0
house 29
jazz 134
latin 32
new age 93
nu jazz 4
pop 175
R&B / soul 0
reggae 58
rock 149
soundtrack 24
tango 9
world 360
Total 1885

It's true that there are many unique terms used, but I chose those 26 terms based on the ones that occurred most. The ones that returned 0 are hip hop/rap and R&B/soul which are combined terms used by iTunes and elsewhere. I can manually clean the ones that have R&B, soul, rap, and hip hop individually.

So based on all that, I think it's still a good idea to eliminate the “Sub genre” search box and keep the Genre search using the dropdown, but have it return all records with that genre value in either Genre or Sub genre fields.
